packages/react-native-web/src/exports/ImageBackground/index.js

```diff
@@ -16,12 +16,12 @@ import Image from '../Image';
 import StyleSheet from '../StyleSheet';
 import View from '../View';
 
-type ImageBackgroundProps = {
+type ImageBackgroundProps = {|
```

Comment:
The change is in order to address a flow error (Sorry, I should have mentioned why I had to change types) 

<details>
<summary>Flow Error</summary>

```
Cannot spread props because inexact props [1] is incompatible with exact `ImageProps` [2]. [incompatible-exact]

   packages/react-native-web/src/exports/ImageBackground/index.js:46:7
              v-----
    46|       <Image
    47|         {...rest}
    48|         ref={imageRef}
    49|         style={[
    50|           {
    51|             // Temporary Workaround:
    52|             // Current (imperfect yet) implementation of <Image> overwrites width and height styles
    53|             // (which is not quite correct), and these styles conflict with explicitly set styles
    54|             // of <ImageBackground> and with our internal layout model here.
    55|             // So, we have to proxy/reapply these styles explicitly for actual <Image> component.
    56|             // This workaround should be removed after implementing proper support of
    57|             // intrinsic content size of the <Image>.
    58|             width,
    59|             height,
    60|             zIndex: -1
    61|           },
    62|           StyleSheet.absoluteFill,
    63|           imageStyle
    64|         ]}
    65|       />
              -^ [1]

References:
   packages/react-native-web/src/exports/Image/index.js:173:3
   173|   ImageProps,
          ^^^^^^^^^^ [2]
```
</details>

Usually it's preferable to use exact types for props

And this is the reason why Image props need to be exact 

<details>
<summary>Flow Error</summary>

```
Error ------------------------------------------------------ packages/react-native-web/src/exports/Image/index.js:413:10
> npm run flow && npm run format && npm run lint && npm run unit --runInBand
Cannot spread props because Flow cannot determine a type for props [1]. object literal [2] is inexact, so it may contain
`ref` with a type that conflicts with `ref`'s definition in props [1]. Try making object literal [2] exact.
[cannot-spread-inexact]

   packages/react-native-web/src/exports/Image/index.js:413:10
   413|   return <BaseImage ref={ref} {...propsToPass} />;
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ [1]

References:
   packages/react-native-web/src/exports/Image/index.js:403:23
                              v
   403|   const propsToPass = {
   404|     ...props,
   405|     // Omit `onLoadStart` because we trigger it in the current scope
   406|     onLoadStart: undefined,
   407|     // Until the current component resolves the request (using headers)
   408|     // we skip forwarding the source so the base component doesn't attempt
   409|     // to load the original source
   410|     source: blobUri ? { ...nextSource, uri: blobUri } : undefined
   411|   };
          ^ [2]


Error ------------------------------------------------------ packages/react-native-web/src/exports/Image/index.js:419:12

Cannot spread props because Flow cannot determine a type for props [1]. `ImageProps` [2] is inexact, so it may contain
`ref` with a type that conflicts with `ref`'s definition in props [1]. Try making `ImageProps` [2] exact.
[cannot-spread-inexact]

   packages/react-native-web/src/exports/Image/index.js:419:12
   419|     return <ImageWithHeaders ref={ref} {...props} />;
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ [1]

References:
   packages/react-native-web/src/exports/Image/index.js:173:3
   173|   ImageProps,
          ^^^^^^^^^^ [2]


Error ------------------------------------------------------ packages/react-native-web/src/exports/Image/index.js:422:10

Cannot spread props because Flow cannot determine a type for props [1]. `ImageProps` [2] is inexact, so it may contain
`ref` with a type that conflicts with `ref`'s definition in props [1]. Try making `ImageProps` [2] exact.
[cannot-spread-inexact]

   packages/react-native-web/src/exports/Image/index.js:422:10
   422|   return <BaseImage ref={ref} {...props} />;
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ [1]

References:
   packages/react-native-web/src/exports/Image/index.js:173:3
   173|   ImageProps,
          ^^^^^^^^^^ [2]



Found 3 errors
```
</details>

It was easier to fix the prop type definition than to put $FlowFixMe on a few lines

Reply:
Got it, thanks for the explanation!